packages/components/src/components/Input/index.tsx

```diff
@@ -52,7 +51,9 @@ export const Input: React.FC<IInputProps> = ({
           <label htmlFor={id}>{props.placeholder}</label>
         </VisuallyHidden>
       ) : null}
-      {label ? <Label htmlFor={id}>{label}</Label> : null}
+      <Label size={2} as="label" htmlFor={id}>
```

Comment:
This is a mixture of styles :)

It would be nicer as 

a) 
```jsx
<Text as="label" size={2} marginBottom={2} htmlFor={id} css={{display: 'block' }}><Text>
```
* it seems we'll need to combine margin with display: block/inline-block quite often. should we add the option to make Text block - <Text block /> or <Text display="block"/>. This property could also be moved all the way up the Element.

or 

b) 
```js
const Label = styled(Text).attrs({as: 'label', size: 2})(css({
  marginBottom: 2,
  display: block
}))

<Label htmlFor={id}></Label>
```

Reply:
Done! Used the first one